packages/plugins/plugin-chess/src/meta.ts

```diff
@@ -9,7 +9,8 @@ export const CHESS_PLUGIN = 'dxos.org/plugin/chess';
 export const meta = {
   id: CHESS_PLUGIN,
   name: 'Chess',
-  description: 'Play chess.',
+  description: `Chess is a simple plugin which allows you to render an interactive chessboard inside of a plank. This chessboard can be used for collaborative chess games, or for practice experiments with the LLM. `,
```

Comment:
avoid the term plank (i.e., don't assume knowledge of internal product details).